orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DisableClusterTask.groovy

```diff
@@ -36,6 +36,16 @@ class DisableClusterTask extends AbstractClusterWideClouddriverTask {
     boolean preferLargerOverNewer = false
   }
 
+  public static class DisableOperation {
+    Integer desiredPercentage
```

Comment:
is there an equivalent clouddriver update that makes disableServerGroup understand this option?